### PR TITLE
Containerize application

### PR DESCRIPTION
**Pull request by believemanasseh**
Closes #28

### CODE REVIEW

File: backend/Pipfile
Comment (zatoa) [inline backend/Pipfile:1]: Just a query, are we using * for versioning?

**Comment by zatoa** [comment]
Formatting code -> https://github.com/SHIV5T3R/CO-DE/issues/67

File: backend/Pipfile
Comment (believemanasseh) [reply backend/Pipfile:1]: Yes. cc @SHIV5T3R

Comment (believemanasseh) [reply backend/Pipfile:1]: I've reverted back to specific versions

**Pull request merged**